src/pages/KimHeeSeongPage/components/poketmon/poketmon.jsx

```
@@ -0,0 +1,29 @@
+import SearchHeader from "../../../../shared/components/SearchHeader";
+import { getPokemonData } from "../Api/poketmonApi";
+import { useState, useEffect } from "react";
+
+export function Poketmon(id) {
+  const [favoriteNum, setFavoriteNum] = useState();
+  const [poketmonData, setPoketmonData] = useState([]);
+
+  const handlePoketSearch = (favoriteNum) => {
+    getPokemonData((id = favoriteNum)).then((data) => {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)